bayesian_changepoint_detection_experiment/03_changepoint_exploration_experiment.py

```diff
@@ -323,9 +332,61 @@ def examine_words_at_timepoint_range(
 fig.savefig(f"{str(figure_output_path)}/2002_changepoint.svg")
 fig.savefig(f"{str(figure_output_path)}/2002_changepoint.png", dpi=500)
 
+plot_values(change_metric_df, change_point_df, tok="2005")
```

Comment:
Its a little hard to see but it looks like either quantification (frequency_ratio or ratio_metric) have a similar change point detection predicted. Though it looks like there is some normalization in the ratio_metric, which I'm not sure if this is significant for which one you select to use.